phonon/update.py

```diff
@@ -88,16 +88,22 @@ def is_expired(self):
         return (current_time > self.hard_expiration or
                 current_time > self.soft_expiration)
 
+    def force_expiry(self):
+        """
```

Comment:
Good point. I'm going to add `self.end_session()` into the `force_expiry()` method so it will actually do what its name implies. 
I don't think we have a use-case for ever setting it to `True` then back to `False`.